package.json

```diff
@@ -23,8 +23,8 @@
     "standard": "^11.0.0"
   },
   "engines": {
-    "node": ">= 7.7.0",
-    "npm": ">= 4.0.0"
+    "node": ">=8.3.0",
+    "npm": ">=5.6.0
```

Comment:
@aps120797 You forgot a `"` here, that's why build is failing.

Reply:
Thanks @knrt10 , fixed it! 😄